### ISSUE TITLE
Provide examples of AJV JSON schema validation with Datasworn 

### ISSUE THREAD

**Issue by rsek** [rsek/datasworn#38]
* show validation of an entire `RulesPackage` object
* show validation primary/embeddable node types individually (and, ideally, for any subschema in `definitions`)
* show how to insert defaults when validating
* provide a config object -- or even a separate package with an AJV dependency, that serves as a friendlier wrapper?
* what should people who only need one major category of schemas (e.g. oracles) do? it's a big schema!

**Comment by martinellison** [comment]
Endorse this, We need executable example so we know how to use this.

**Comment by tbsvttr** [comment]
> Endorse this, We need executable example so we know how to use this.

I agree: it would be important to have better documentation to be able to contribute.
